new-log-viewer/src/components/DropFileContainer/index.css

```diff
@@ -0,0 +1,31 @@
+.drop-file-container {
+    width: 100%;
+    height: 100%;
+    position: relative;
+}
+
+.drop-file-children {
+    width: 100%;
+    height: 100%;
+}
+
+.hover-mask {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(2, 88, 168, 0.2);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 10;
+}
+
+.hover-message {
+    padding: 8px;
+    color: #616161;
+    font-size: 14px;
+    font-family: Roboto, sans-serif;
+    background-color: #f3f3f3;
+    z-index: 11;
```

Comment:
Any objection if we put up some global variables in `index.css` so that we can see all `z-index` values in one place? Asking since this might not be good for modulization and having a fallback value in DropFileContainer/index.css doesn't make too much sense either.

Reply:
Yup, what I had in mind was that we could define the variables in the global spreadsheet and use it here like `var(--global-variable-name)`.

Reply:
Done in the latest commits.